ObjectiveGit/GTSignature.h

```diff
@@ -42,10 +42,10 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, copy, nullable) NSString *email;
 
 /// The time when the action happened.
-@property (nonatomic, readonly, strong) NSDate *time;
+@property (nonatomic, readonly, strong, nullable) NSDate *time;
```

Comment:
IIRC it's done that way because libgit2 might fail to parse an object's headers (in some degenerate-odb cases), hence allowing some of them to be null.